model/model.go

```diff
@@ -1,5 +1,7 @@
 package model
 
+import "github.com/joshprzybyszewski/cribbage/auth"
```

Comment:
We should avoid importing auth into `model`. We want `model` to remain as purely as possible a "model"

Reply:
Sure... `Credentials` can be defined in `model` and imported into `auth`, is that what you mean?

Reply:
It seems like the `type Credentials struct` should exist in the `auth/` package, and then we should import model there if we still need it (probably just for `model.PlayerID`)

Reply:
We need to persist the username and hashed password, so I think we'd need those to exist on the model? Or I guess we could import both `auth` and `model` into `persistence` to persist everything